roles/gerrit/tasks/gerrit.yml

```diff
@@ -0,0 +1,86 @@
+# vim:ft=ansible:
+---
+- hosts: gluster_nodes
+  vars:
+    gerrit_url: review.gluster.org
+    gerrit_email: review@dev.gluster.org
+    gerrrit_user: review
+    gerrit_user_home: /review
+    gerrit_home: review.gluster.org
+    gerrit_db_host: localhost
+    gerrit_db_user: gerrit
+    gerrit_db_name: gerrit
+    gerrit_db_pass: hunter2
+    gerrit_user: review
+    github_client_id: 123
+    github_client_secret: 456
+    bugzilla_user: bugzilla-bot@gluster.org
+    bugzilla_pass: hunter2
+    gerrit_replication:
+    - repo: glusterfs
+      github_user: gluster
+      keyfile: glusterfs-github
+    - repo: glusterfs-specs
+      github_user: gluster
+      keyfile: glusterfs-spec-github
+    - repo: glusto-tests
+      github_user: gluster
+      keyfile: glusto-tests-github
+  tasks:
+  - name: Install Java and python-bugzilla
+    package: state=present name={{ item }}
+    with_items:
+    - java-1.8.0-openjdk
+    - python-bugzilla
+
+  - name: Create a user for review
+    user:
+      name: "{{gerrit_user}}"
+      home: "{{gerrit_user_home}}"
+
+  - name: Setup the replication.config file
+    template:
+      src: replication.config
+      dest: "{{gerrit_user_home}}/{{gerrit_home}}/etc/replication.config"
+      owner: "{{gerrit_user}}"
+      mode: 0755
+
+  - name: Setup the SSH config file
+    template:
+      src: ssh.config
+      dest: "{{gerrit_user_home}}/.ssh/config"
+      owner: "{{gerrit_user}}"
+      mode: 0755
+
+  - name: Setup the replication keys
+    copy:
+      src: "{{item.keyfile}}"
+      dest: "{{gerrit_user_home}}.ssh/{{item.keyfile}}"
+      owner: "{{gerrit_user}}"
+      mode: 0700
+    with_items: gerrit_replication
+
+  - name: Cron to delete old log files
+    cron:
+      name: "Clean logs"
+      special_time: "weekly"
+      job: "find {{gerrit_user_home}}/{{gerrit_home}}/logs -mtime +15 -delete"
+      user: {{gerrit_user}}
+
+  - name: Setup python-bugzilla
+    command: "bugzilla login {{bugzilla_user}} {{bugzilla_pass}}
+    no_log: True
```

Comment:
This would generate a notification on each run, I think it would be better to run only if needed. I guess it create a file, we should add creates=somefilesomewhere to prevent it from being run each time